pkg/util/grpcutil/grpc_util.go

```diff
@@ -50,6 +50,7 @@ func IsClosedConnection(err error) bool {
 		grpc.Code(err) == codes.Unavailable ||
 		grpc.ErrorDesc(err) == grpc.ErrClientConnClosing.Error() ||
 		strings.Contains(err.Error(), "is closing") ||
+		strings.Contains(err.Error(), "use of closed connection") ||
```

Comment:
Looks like there's some duplication between this and another check in `rpc/context_test.go`. Could you merge the two? See `gRPCErrorsRegex` in that file.